site/docs/foundations/spacing.mdx

```diff
@@ -13,16 +13,16 @@ Every space token within the system is a multiple of `--salt-spacing-100`. The n
 
 | Token                | Formula | High Density (HD) | Medium Density (MD) | Low Density (LD) | Touch Density (TD) |
 | -------------------- | ------- | ----------------- | ------------------- | ---------------- | ------------------ |
-| `--salt-spacing-25`  | 0.25x   | 1px               | 2px                 | 3px              | 4px                |
-| `--salt-spacing-50`  | 0.50x   | 2px               | 4px                 | 6px              | 8px                |
-| `--salt-spacing-75`  | 0.75x   | 3px               | 6px                 | 9px              | 12px               |
-| `--salt-spacing-100` | 1.00x   | 4px               | 8px                 | 12px             | 16px               |
-| `--salt-spacing-150` | 1.50x   | 6px               | 12px                | 18px             | 24px               |
-| `--salt-spacing-200` | 2.00x   | 8px               | 16px                | 24px             | 32px               |
-| `--salt-spacing-250` | 2.50x   | 10px              | 20px                | 30px             | 40px               |
-| `--salt-spacing-300` | 3.00x   | 12px              | 24px                | 36px             | 48px               |
-| `--salt-spacing-350` | 3.50x   | 14px              | 28px                | 42px             | 56px               |
-| `--salt-spacing-400` | 4.00x   | 16px              | 32px                | 48px             | 64px               |
+| `--salt-spacing-25`  | 0.25x   | 1                 | 2                   | 3                | 4                  |
```

Comment:
It's not clear that these are px values now, should px be added in the headers or as a caption?

Reply:
Great point, will add that in.